src/Compilers/Shared/DesktopBuildClient.cs

```diff
@@ -157,12 +157,12 @@ protected override int RunLocalCompilation(List<string> arguments, string client
                 }
             }
 
-            return null;
+            return Task.FromResult<BuildResponse>(null);
```

Comment:
I swear we should write a rule that flags returning `null` from a `Task` based method :frowning: